docs/ad/index.rst

```diff
@@ -0,0 +1,42 @@
+.. include:: ad_attention.inc
+
+.. _ad:
+
+==============================
+Analysis & Diagnostic Toolkits
+==============================
+
+Analyses and diagnostics go hand-in-hand, but have subtle differences.  Thus,
```

Comment:
Eventually we'll want to have analyses related to simulations, for example, so we'll eventually want to change the wording a bit.  But, this wording is fine for now.